assets/src/edit-story/utils/calculateFitTextFontSize.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Internal dependencies
+ */
+import { MIN_FONT_SIZE, MAX_FONT_SIZE } from '../constants';
+
+/**
+ * Calculates font size that fits to the text element based on the element's size.
+ * Replicates amp-fit-text's logic in the editor.
+ *
+ * @see https://github.com/ampproject/amphtml/blob/e7a1b3ff97645ec0ec482192205134bd0735943c/extensions/amp-fit-text/0.1/amp-fit-text.js
+ *
+ * @param {Object} measurer       HTML element.
+ * @param {number} expectedHeight Maximum height.
+ * @param {number} expectedWidth  Maximum width.
+ *
+ * @return {number|boolean} Calculated font size. False if calculation wasn't possible.
+ */
+export const calculateFitTextFontSize = ( measurer, expectedHeight, expectedWidth ) => {
```

Comment:
@dvoytenko This is still WIP PR but IIRC then you and @pbakaus mentioned potentially a different way for measuring font size based on the room there is.

Pinging you both here if you had something specific in mind instead of this approach (which replicates what amp-fit-text does).

Reply:
Merging as-is, we can come back to it later if there should be a better alternative.